lib/ansible/modules/windows/win_nssm.ps1

```diff
@@ -544,16 +545,16 @@ Function Nssm-Start
 
     if ($LastExitCode -ne 0)
     {
-        Set-Attr $result "nssm_error_cmd" $cmd
-        Set-Attr $result "nssm_error_log" "$results"
+        $result.nssm_error_cmd = $cmd
+        $result.nssm_error_log = "$results"
         Throw "Error starting service ""$name"""
     }
 
     switch ($currentStatus)
     {
         "SERVICE_RUNNING" { <# Nothing to do #> }
         "SERVICE_STOPPED" { Nssm-Start-Service-Command -name $name }
-        
+
```

Comment:
Nope, just a cleanup of trailing whitespaces. My editor makes them quite obvious, so I remove them.